test/input-test.html

```diff
@@ -205,6 +205,11 @@
           });
         });
 
+        it(`should reflect to attribute when readonly property is set`, () => {
```

Comment:
No need for template literal here, please use regular single quotes instead.
```suggestion
        it('should reflect to attribute when readonly property is set', () => {
```

Reply:
done